src/machines/java/com/enderio/machines/common/menu/GhostMachineSlot.java

```diff
@@ -18,7 +18,7 @@ public GhostMachineSlot(MachineInventory itemHandler, int index, int xPosition,
         MachineInventoryLayout layout = itemHandler.getLayout();
         if (layout.canInsert(index) || layout.canExtract(index))
             throw new RuntimeException("Ghost slot can be externally modified!!");
-        if (!layout.guiCanInsert(index) || !layout.guiCanExtract(index))
+        if (!layout.guiCanInsert(index))
```

Comment:
Why was this change again @ferriarnus, I remember you explained it but it slips my mind

Reply:
I don't know what this exactly does but trying to open the container in-game without that line causes the exception(thrown under that line) to arise.